src/Aspire.Hosting.Azure/Extensions/AzureSearchExtensions.cs

```diff
@@ -26,4 +29,54 @@ public static IResourceBuilder<AzureSearchResource> AddAzureSearch(this IDistrib
                 .WithParameter(AzureBicepResource.KnownParameters.PrincipalType)
                 .WithManifestPublishingCallback(resource.WriteToManifest);
     }
+
+    /// <summary>
+    /// Adds an Azure AI Search service resource to the application model.
+    /// </summary>
+    /// <param name="builder">The builder for the distributed application.</param>
+    /// <param name="name">The name of the Azure AI Search resource.</param>
+    /// <param name="configureResource">Callback to configure the Azure AI Search resource.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{AzureSearchConstructResource}"/>.</returns>
+    public static IResourceBuilder<AzureSearchConstructResource> AddAzureConstructSearch(
```

Comment:
I think these would work better for customers if we called it `AddAzureSearchConstruct` because then it'll appear in intellisense next to the vanilla `AddAzureSearch`.

Reply:
We'll need to polish all of these depending on what our decision is around CDK exposure.